test/RecipeRepository-test.js

```diff
@@ -1,8 +1,32 @@
-import { expect } from 'chai';
-import RecipeRepository from '../src/classes/RecipeRepository';
+import { expect } from "chai";
+import RecipeRepository from "../src/classes/RecipeRepository";
+import recipes from "../src/data/recipes";
+// const recipes = require("..src/data/recipes");
 
-describe('Recipe', () => {
-  it('Should be a function', () => {
-    expect(RecipeRepository).to.be.a('function');
+describe("Recipe", () => {
+  let data;
+  let recipeRepo;
+  beforeEach(() => {
+    data = recipes;
+    recipeRepo = new RecipeRepository(data);
   });
-})
\ No newline at end of file
+  it("Should be a function", () => {
```

Comment:
Great work! Maybe a future update could be changing some of these tests and the RecipeRepository class to take in Ingredient and Recipe classes.